packages/api-key-stamper/src/__tests__/elliptic-curves-test.ts

```diff
@@ -0,0 +1,80 @@
+import { test, expect } from "@jest/globals";
+import { pointDecode } from "../tink/elliptic_curves";
+import {
+  uint8ArrayFromHexString,
+  uint8ArrayToHexString,
+} from "@turnkey/encoding";
+
+test("pointDecode -> uncompressed invalid", async function () {
+  // Invalid uncompressed key (the last byte has been changed)
+  const uncompPubKey =
+    "0400d2eb47be2006c29db5fe9941dd686d19ddeea85a0328894f08091f6b5be9b366c4872345594c12a7f7c47c62dd8074542934820fce5ee0ddc55d6d1d8dd313";
+  expect(() => pointDecode(uint8ArrayFromHexString(uncompPubKey))).toThrow(
+    "invalid uncompressed x and y coordinates"
+  );
+});
+
+test("pointDecode -> uncompressed valid", async function () {
+  // Valid uncompressed public key with 00 as the first bit (test against 'x' field of JWK getting truncated)
+  const uncompPubKey =
+    "0400d2eb47be2006c29db5fe9941dd686d19ddeea85a0328894f08091f6b5be9b366c4872345594c12a7f7c47c62dd8074542934820fce5ee0ddc55d6d1d8dd312";
+  const jwk = pointDecode(uint8ArrayFromHexString(uncompPubKey));
+  expect(jwk.x).toBeDefined();
+  expect(jwk.y).toBeDefined();
+
+  // Convert x value to make sure it matches first half of uncompressed key WITHOUT truncating the first 0 bit
```

Comment:
This is great! If you'd like, you could also try importing the resulting JWK as a sanity check as well (e.g. the `crypto.subtle.importKey...` codepath that happens in `packages/sdk-browser/src/utils.ts`).

Note: if you do go down this path, you might need to import `crypto` to have it defined in the test setting:

```
Object.defineProperty(globalThis, "crypto", {
  value: {
    getRandomValues: (arr) => crypto.randomBytes(arr.length),
    subtle: crypto.subtle,
  },
});
```

Can use `packages/api-key-stamper/src/__tests__/utils-test.ts` in [this diff](https://github.com/tkhq/sdk/pull/322/files#diff-b3f5d0933ac8568c7a900ac4c1b3a23afa74499ec22d8b6d0d7e44d200192b58) as a reference

Reply:
Totally fine if you think this is overkill given some of the manual validation you're doing here

Reply:
Thanks for the suggestion here!! I'll leave the manual validation as is for right now so we can get this over the line in a timely manner